### DISCUSSION TITLE
Is Codesandbox down today Cant start Sandbox?

### DISCUSSION THREAD

**Discussion by erikswed** [codesandbox/codesandbox-client#5479] [General]
Windows 10 here, This morning in Google Chrome everything was ok as it has been for month. Suddenly the sandbox refuse to run with no warning nothing look at image all I get is white and sometimes with some pink undertermen ..

I try hard reset ctrl+f5, I try clear cookies, I try restart laptop ASUS  ROG
Any ide?
![image](https://user-images.githubusercontent.com/3285249/107936974-9835e580-6f83-11eb-8c7f-0b594c1b02a5.png)
![image](https://user-images.githubusercontent.com/3285249/107943145-8ad12900-6f8c-11eb-8456-8b64a97a090f.png)

**Comment by erikswed** [comment]
This is one of the Sandbox that I cant start; it's the one in the image with pink undertermen.. It's also refuse to run in Edge Browser

https://codesandbox.io/s/modest-thompson-j9dw3?file=/package.json

**Comment by erikswed** [comment]
This happens only when I use the [Google tabs](https://blog.google/products/chrome/manage-tabs-with-google-chrome/). In the same Google Browser if I open the CodeSandbox in a tab that is not inside a Google tab it works!

**Comment by SaraVieira** [comment]
Hey!

Is this still happening? Does it happen on an incognito window?

**Reply by erikswed** [reply]
yes and no, it's sporadic I created a bug report at chromium also
https://bugs.chromium.org/p/chromium/issues/detail?id=1178752#c1
